.gitlab-ci.yml

```diff
@@ -1,10 +1,18 @@
 variables:
-  CI_IMAGE: registry.ddbuild.io/images/docker:20.10.13
-  RESTORE_CACHE_ATTEMPTS: 2
+  # https://github.com/DataDog/cloudops/tree/master/AWS/build-stable/ci/dd-trace-demo
+  CI_PROJECT_NAME: 'dd-trace-demo'
```

Comment:
Notes: needs to match https://github.com/DataDog/cloudops/blob/master/AWS/build-stable/ci/opentelemetry-collector-contrib/iam.tf#L3